### DISCUSSION TITLE
Marpit

### DISCUSSION THREAD

**Discussion by itsmeadarsh2008** [catppuccin/catppuccin#1847] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Maybe

**Comment by itsmeadarsh2008** [comment]
When?

**Comment by sgoudham** [comment]
Hey :wave:,

I've closed this discussion since the original post doesn't include any links on how to theme the application or link to any source code of other themes being applied to the application.

Would be more than happy to re-open once the post is updated with the relevant links or instructions on how to theme it. 

Thanks!
